### DISCUSSION TITLE
delete best wpm

### DISCUSSION THREAD

**Discussion by abhaychaturvedii** [monkeytypegame/monkeytype#4161] [Ideas] [answered]
we don't have a option to delete our best wpm 

one of my friend uses my pc and he has best wpm in my id and I want to delete it 

![image](https://user-images.githubusercontent.com/55993241/230770629-13a8243f-a913-46d3-aaa1-818f6eeb1a32.png)

like here in 30 second , 65 wpm , i want to delete it , so do we have a option for this ?

**Comment by abhaychaturvedii** [comment]
nobody has answered it yet ?

**Comment by unrealapex** [comment, marked as answer]
You can. The setting is `reset personal bests`, it is in the Danger Zone section of the settings page. Please make a reasonable effort to find an answer to a question yourself before you contact the maintainers of open source projects. It wastes less of their time.

**Comment by abhaychaturvedii** [comment]
thanks man , got it :)
